### DISCUSSION TITLE
1.13/1.12など低いversionの適性について

### DISCUSSION THREAD

**Discussion by crh5a-5128** [kous500/CurveBuilding#12] [Q&A] [answered]
将来、1.13/1.12などの低いversion適用可能なpluginを開発する予定がありますか

**Comment by wacch** [comment]
Since the creator of this project is no longer active and maintenance has become difficult, we cannot hopefully work on the implementation of this request.
If anyone is willing to help implement this feature, please open a PullRequest.

**Comment by kous500** [comment, marked as answer]
If there are many version requests, development may begin.
